diff --git a/node_modules/editpix/src/image-manager.js b/node_modules/editpix/src/image-manager.js
index ac9a368..0e7158f 100644
--- a/node_modules/editpix/src/image-manager.js
+++ b/node_modules/editpix/src/image-manager.js
@@ -175,7 +175,8 @@ export default class ImageManager {
      * 
      */
     getImageType(imageSrc) {
-        const result = imageSrc.match(/(jpg|jpeg|png|gif|bmp|webp)/)[0];
+        //const result = imageSrc.match(/(jpg|jpeg|png|gif|bmp|webp)/)[0];
+        const result = "png";
         if (result === "jpg")
             return "jpeg";
         return result;
